wallet/sr25519/address_test.go

```diff
@@ -42,17 +41,3 @@ func TestAddress_SS58Addr(t *testing.T) {
 		}
 	}
 }
-
-// TestAddress_ZeroCmp tests that the zero address is strictly
-// smaller than all other addresses.
-func TestAddress_ZeroCmp(t *testing.T) {
```

Comment:
Why exactly is this test removed?

Reply:
Because Cmp is no longer part of the interface.